abilities/hre/emergency-repairs-1.json

```diff
@@ -0,0 +1,54 @@
+{
+  "id": "emergency-repairs-1",
+  "baseId": "emergency-repairs",
+  "type": "ability",
+  "name": "Emergency Repairs",
+  "pbgid": 2031580,
+  "attribName": "influence_auto_repair_hre",
+  "age": 1,
+  "civs": [
+    "hr"
+  ],
+  "description": "Building repairs itself by 150 health every 1 second for 20 seconds.",
+  "classes": [],
+  "displayClasses": "",
+  "costs": {
+    "food": 0,
+    "wood": 0,
+    "stone": 0,
+    "gold": 0,
+    "total": 0,
+    "popcap": 0
+  },
+  "producedBy": [
```

Comment:
Hmmm, don't we have class 'building' ? Maybe we should haha

Reply:
producedBy should be town-center as influence of TC grants ER.
effects should have healingRate apply to class buildings

Thoughts?

Reply:
I'm really thinking this in terms of 'in game, in the UI, where do I click the button to make/trigger it'. Being in range of the TC would be a requirement, just like 'targeting a unit ' for certain abilities is a requirement.

Reply:
I see what you are saying.  Do you prefer "building" or "town-center"?
![image](https://github.com/aoe4world/data/assets/3922144/ddcf3bf5-184e-440d-a7d6-7f1d3af82a73)
![image](https://github.com/aoe4world/data/assets/3922144/e5005def-9313-455f-8ab6-d9b38a5c82fd)

Reply:
Hmm this one is extremely edge-casy, but let's keep all the building in the `producedBy` like it currently is.